locales/it/index.json

```
@@ -252,6 +252,15 @@
         "subtitle": "Collegati a internet per continuare ad usare tutti i servizi di app IO.",
         "action": "Riprova"
       },
+      "statusMessage": {
+        "message": "Nessuna connessione",
+        "action": "Cosa posso fare?"
+      },
+      "connectionRestored": "Connessione ripristinata.",
+      "bottomSheet": {
+        "title": "Nessuna connessione",
+        "content": "## Cosa posso fare? \n\nQuando sei offline, alcuni servizi dell’app potrebbero non funzionare. \n\n Documenti su IO funziona anche offline (se hai attivato la funzionalità), ma la validità dei documenti è aggiornata all’ultimo accesso in app con connessione a una rete. \n\n Collegati a internet e ricarica l’app per tornare ad usare tutti i servizi di IO."
```

According to the Figma design the heading should be smaller an H6
```suggestion
        "content": "###### Cosa posso fare? \n\nQuando sei offline, alcuni servizi dell’app potrebbero non funzionare. \n\n Documenti su IO funziona anche offline (se hai attivato la funzionalità), ma la validità dei documenti è aggiornata all’ultimo accesso in app con connessione a una rete. \n\n Collegati a internet e ricarica l’app per tornare ad usare tutti i servizi di IO."
```